packages/core-components/src/lib/unsorted/ui/Modal.svelte

```diff
@@ -0,0 +1,62 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	import CloseIcon from '../icons/CloseIcon.svelte';
```

Comment:
I mentioned this in another PR, and I don't know if we have a solid answer for this yet, but we want to normalize on an icon library:

https://github.com/evidence-dev/evidence/pull/955#discussion_r1232621678

Reply:
I'm going to open a pull request today or tomorrow adding the icon library, once that is merged you can merge it into your branch and leverage that.